igor/igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/EchoConfig.groovy

```diff
@@ -18,7 +18,7 @@ package com.netflix.spinnaker.igor.config
 
 import com.netflix.spinnaker.config.OkHttp3ClientConfiguration
 import com.netflix.spinnaker.igor.IgorConfigurationProperties
-import com.netflix.spinnaker.igor.history.EchoConverterFactory
+import com.netflix.spinnaker.kork.retrofit.util.CustomConverterFactory
```

Comment:
Could we have this refactor in a separate commit please?

Reply:
Agreed. Will do.